app/apis/hackney_api/repairs_client.rb

```diff
@@ -87,15 +87,16 @@ def get_work_order_notes(reference)
       )
     end
 
-    def post_work_order_note(work_order_reference, text)
+    def post_work_order_note(work_order_reference, text, created_by_email)
       value = request(
         http_method: :post,
         endpoint: "#{API_VERSION}/notes",
         headers: {"Content-Type" => "application/json-patch+json"},
         params: {
           objectKey: "uhorder",
           objectReference: work_order_reference,
-          text: text
+          text: text,
+          lbhemail: created_by_email
```

Comment:
Is the API requiring it to be all lowercase? Everywere else it's lhbEmail.

Reply:
That's what Bukky sent it to me as

Reply:
and it worked when I created a note

Reply:
ok I'm not gonna complain about this :sparkles:  :sparkles:  :sparkles: